import "node.proto";
import "item.proto";
import "resource.proto";

package SanProto;

//资源点增产
message IncreaseReq {
  required uint64 user_id = 1;
  required NodeInfo node = 2;
  optional ItemInfo item = 3;
}
message IncreaseRes {
  required int32 status = 1;
  optional ResourceInfo resource = 2;
}
